config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
+  get 'users/new'
+	root 'static_pages#home'
+	get  '/help', 	to: 'static_pages#help'
+	get  '/about',	to: 'static_pages#about'
+	get  '/contact',to: 'static_pages#contact'
+	get  '/signup',  to: 'users#new'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 end
```

Comment:
Same comment for this file like as the previous comment and remove the comments.